### PR TITLE
Add Default Quota Calculation

### PR DESCRIPTION
**Pull request by ayuishii**
This adds a section for default quota calculation to design discussion.

### CODE REVIEW

File: explainer.md [hunk 1]
```diff
@@ -827,6 +828,16 @@ facts were considered by our decision process.
   [WriteOptions.sync option](https://github.com/google/leveldb/blob/master/doc/index.md#synchronous-writes).
 
 
+### Default Quota
+
+We propose the default quota for a storage bucket to match the origin quota.
+This may seem unintuitive to have quota for a storage bucket be so
+large, disconnecting further from available disk space.
+However having anything under 100% of the origin quota will become a
+constrain to developers, disincentivizing the use of buckets.
```
Comment (pwnall) [inline explainer.md:837]: constrain (v) -> constraint (n)

File: explainer.md [hunk 2]
```diff
@@ -827,6 +828,16 @@ facts were considered by our decision process.
   [WriteOptions.sync option](https://github.com/google/leveldb/blob/master/doc/index.md#synchronous-writes).
 
 
+### Default Quota
+
+We propose the default quota for a storage bucket to match the origin quota.
+This may seem unintuitive to have quota for a storage bucket be so
+large, disconnecting further from available disk space.
+However having anything under 100% of the origin quota will become a
```
Comment (pwnall) [inline explainer.md:836]: Let's add "We think" to this sentence and the next one, to soften them.

As far as I know, other browsers (especially Firefox) plan to take a different approach here. They currently have much smaller quotas (1-5GB / origin). I think their default bucket will match their current quotas, and they'll grant developers additional quota as they create additional buckets.

There is some merit to this approach -- the browser is in a slightly better position to handle storage pressure gracefully, as it's guaranteed to have O(number of GB used) buckets to choose from. We (Chrome) don't think this is the best approach for the ecosystem, so we won't be pursuing it. But we (the Buckets API designers) should come up with a design that allows this approach, unless we become convinced that it's completely wrong / harmful to the ecosystem.

**Review by pwnall — CHANGES_REQUESTED** [review]
Sorry for the really long review time!

File: explainer.md (hunk 2 above)
Comment (ayuishii) [reply explainer.md]: This makes sense, thanks for this explanation!

**Review by ayuishii — COMMENTED** [review]
Thanks for the review!

File: explainer.md [hunk 3]
```diff
@@ -827,6 +828,20 @@ facts were considered by our decision process.
   [WriteOptions.sync option](https://github.com/google/leveldb/blob/master/doc/index.md#synchronous-writes).
 
 
+### Default Bucket Quota
+
+A default quota will be assigned to every storage bucket that is created
+with `openOrCreate()` without a `quota` policy. The behavior of the
+default bucket quota will be user agent specific.
+
+In Chrome, we plan to have the default quota for a storage bucket to
```
Comment (pwnall) [inline explainer.md:837]: Let's avoid saying "we" to mean Chrome engineers. This doc will hopefully gain non-Chrome authors.

In Chrome, we plan to have -> Chrome plans to have
we think -> the Chrome team thinks

**Review by pwnall — APPROVED** [review]
LGTM w/ comment addressed. Thank you for the revisions!

File: explainer.md (hunk 3 above)
Comment (ayuishii) [reply explainer.md]: Sorry :( I keep on forgetting. 
Thanks for catching this.

**Pull request merged**